tests/ui/test_ui_tools.py

```diff
@@ -309,15 +309,15 @@ def test_read_message(self, mocker, msg_box):
         msg_view.read_message()
         assert msg_view.update_search_box_narrow.called
```

Comment:
@neiljp Since this PR contains changes along similar lines (being renaming functions for clarity),
Do you want to rename `test_init` in `Class TestMessageBox` to something more appropriate like `test_general_message` (Line number 848)?

Reply:
If you're meaning renaming `test_init`, I think that there is certainly work that can be done to simplify/clarify some of the test cases, though I think we can be more specific than `general_message`. I wouldn't say this belongs in this PR, but perhaps you meant this as a side question/comment, rather than a review?

Reply:
Right! A quick `git grep` shows a lot of tests with the name `test_init`. Keeping this separate is what I meant. I should not have posted it as a review. Apologies for that.